src/api/config/mod.rs

```diff
@@ -397,6 +407,36 @@ impl Config {
       }
     }
 
+    if let Some(level_idx) = config.level_idx {
+      if level_idx > 31 {
+        return Err(LevelUndefined);
+      }
+      if level_idx < 31 {
```

Comment:
Not sure if this is intentional for style, but that conditional is unnecessary.

Reply:
it is ignoring the level_idx == 31 case

Reply:
Oh right my bad